config/initializers/omniauth.rb

```diff
@@ -64,4 +68,40 @@
     saml_options[:allowed_clock_drift] = ENV['SAML_ALLOWED_CLOCK_DRIFT'] if ENV['SAML_ALLOWED_CLOCK_DRIFT']
     config.omniauth :saml, saml_options
   end
+
+  # OpenID Connect Strategy
+  if ENV['OIDC_ENABLED'] == 'true'
+    oidc_options = {}
+    oidc_options[:redirect_at_sign_in] = ENV['OAUTH_REDIRECT_AT_SIGN_IN'] == 'true' #OPTIONAL
```

Comment:
Drop that line, `ENV['OAUTH_REDIRECT_AT_SIGN_IN']` / `cas_options[:redirect_at_sign_in]` has been removed, in favor of `ENV['OMNIAUTH_ONLY']` which is handled independently